plugin/src/main/java/org/openrewrite/gradle/RewritePlugin.java

```diff
@@ -59,15 +59,14 @@ public void apply(Project project) {
         JavaPluginConvention javaConvention = project.getConvention().getPlugin(JavaPluginConvention.class);
         SourceSetContainer sourceSets = javaConvention.getSourceSets();
 
-        // Fix is meant to be invoked manually and so is not made a dependency of any existing task
         Task rewriteFixAll = tasks.create("rewriteFix",
                 taskClosure(task -> {
                     task.setGroup("rewrite");
                     task.setDescription("Apply the active refactoring recipes to all sources");
                 })
         );
 
-        Task rewriteWarnAll = tasks.create("rewriteWarn", taskClosure(task -> {
+        Task rewriteDryRunAll = tasks.create("rewriteDryRun", taskClosure(task -> {
```

Comment:
Let's keep the old "warn" name working for now, just in case someone uses it. Create another task named rewriteWarn that depends on `rewriteDryRun`. We'll leave the aliases in for a few releases

Reply:
Just to clarify: are we just aliasing `rewriteWarn`, or do we also need to alias `rewriteWarn{SourceSet}`, e.g. users running `rewriteWarnTest`, `rewriteWarnMain`, etc.?

Reply:
Probably just `rewriteWarn`. I think it unlikely that anyone will have bothered with the more granular per-sourceSet tasks

Reply:
Just for good measure I included the specific source sets, too.